swastika-mtv.md

```diff
@@ -0,0 +1,5 @@
+## Mountain View, CA
+
+### Places to eat
```

Comment:
I will be in Mountain View in a few weeks! Any recommendations for great Italian @swastika ?